nukebuild/_build.csproj

```diff
@@ -2,15 +2,15 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
```

Comment:
This TFM was failing for me on several machines with .NET Core SDK 3.1 installed. Could we use `netcoreapp3.1` instead?

Reply:
I guess so, let me try.